package.json

```diff
@@ -12,73 +12,84 @@
     "config": "node .esbuild/helpers/config.js",
     "prepare": "npm run config && npm run build",
     "build": "node esbuild.js",
+    "lint": "npm run lint:js && npm run lint:styles",
```

Comment:
please update prepare/build so config is updated on each build (just in case tokens is changed and new palettes added)
```
    "prepare": "npm run build",
    "build": "npm run config && node esbuild.js",
```